tensorflow-framework/src/main/java/org/tensorflow/framework/initializers/RandomNormal.java

```diff
@@ -31,13 +30,12 @@
  *     RandomNormal&lt;TFloat32, TFloat32&gt; initializer =
  *              new org.tensorflow.framework.initializers.RandomNormal&lt;&gt;(tf, seed);
  *     Operand&lt;TFloat32&gt; values =
- *              initializer.call(tf.constant(Shape.of(2,2)), TFloat32.DTYPE);
+ *              initializer.call(tf.constant(Shape.of(2,2)), TFloat32.class);
  * </pre>
  *
  * @param <T> The TType for the call operation
- * @param <U> The TNumber for the call operation
  */
-public class RandomNormal<T extends TType, U extends TNumber> extends BaseInitializer<T> {
+public class RandomNormal<T extends TNumber> extends BaseInitializer<T> {
```

Comment:
`TFloating`?

Reply:
same thing as https://github.com/tensorflow/java/pull/174#discussion_r547489320

Reply:
[According to](https://github.com/tensorflow/java/pull/174#discussion_r547550405) @JimClarke5 , this one should now be restricted to floating-points only, I'll make the change